### PR TITLE
Telegraf v1.32.0

### PR DESCRIPTION
**Pull request by srebhan**
Closes #

This PR documents missing v1.31.x releases, adds Telegraf v1.32.0 and tags this release as the latest. It also fixes the `telegraf-plugins.yml` list of plugins by adding missing plugins and fix typos in plugin `id`s.

- [x] Rebased/mergeable

### CODE REVIEW

File: content/telegraf/v1/release-notes.md [hunk 1]
```diff
@@ -11,6 +11,214 @@ menu:
...
+
+### Important Changes
+
+- This release contains a logging overhaul as well as some new features for
+  logging (see PRs [#15556](https://github.com/influxdata/telegraf/pull/15556),
+  [#15629](https://github.com/influxdata/telegraf/pull/15629),
+  [#15677](https://github.com/influxdata/telegraf/pull/15677),
+  [#15695](https://github.com/influxdata/telegraf/pull/15695) and
+  [#15751](https://github.com/influxdata/telegraf/pull/15751)).
+  As a consequence the redunant `logtarget` setting is deprecated, `stderr` is
```
Comment (jstirnaman) [inline content/telegraf/v1/release-notes.md:24]:
```suggestion
  As a consequence, the redundant `logtarget` setting is deprecated. `stderr` is
```

File: content/telegraf/v1/release-notes.md [hunk 2]
```diff
@@ -11,6 +11,214 @@ menu:
...
+  used if no `logfile` is provided, otherwise messages are logged to the given
+  file. For using the Windows `eventlog` set `logformat = "eventlog"`!
```
Comment (jstirnaman) [inline content/telegraf/v1/release-notes.md:26]:
```suggestion
  file. To use Windows `eventlog`, set `logformat = "eventlog"`.
```

File: content/telegraf/v1/release-notes.md [hunk 3]
```diff
@@ -11,6 +11,214 @@ menu:
...
+- This release contains a change in json_v2 parser config parsing -
+  if the config is empty (not define any rules), initialization will fail
+  (see PR [#15844](https://github.com/influxdata/telegraf/pull/15844)).
```
Comment (jstirnaman) [inline content/telegraf/v1/release-notes.md:29]:
```suggestion
- This release contains a change in json_v2 parser config parsing: if the config is empty (doesn't define any rules), initialization will fail
  (see PR [#15844](https://github.com/influxdata/telegraf/pull/15844)).
```

File: content/telegraf/v1/release-notes.md [hunk 4]
```diff
@@ -11,6 +11,214 @@ menu:
...
+- This release contains a feature for a disk-backed metric buffer under the
+  `buffer_strategy` agent config (see
+  PR [#15564](https://github.com/influxdata/telegraf/pull/15564)).
+  Please note, this feature is **experimental**, please give it a test and
+  report any issues you encounter.
+
```
Comment (jstirnaman) [inline content/telegraf/v1/release-notes.md:35]:
```suggestion
  _This feature is **experimental**. Please report any issues you encounter while using it._

```

File: data/telegraf_plugins.yml [hunk 5]
```diff
@@ -2548,6 +2572,15 @@ output:
     introduced: 0.1.9
     tags: [linux, macos, windows, data-stores]
 
+  - name: Parquet
+    id: parquet
+    description: |
+      This plugin writes metrics to parquet files. By default, the parquet
+      output groups metrics by metric name and write those metrics all to the
+      same file. If a metric schema does not match then metrics are dropped.
```
Comment (jstirnaman) [inline data/telegraf_plugins.yml:2580]:
```suggestion
      This plugin writes metrics to parquet files. By default, the parquet
      output groups metrics by metric name, and then writes the grouped metrics to a single file. If a metric schema doesn't match, the metrics are dropped.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
A few small fixes. Thanks!

File: data/telegraf_plugins.yml (hunk 5 above)
Comment (srebhan) [reply data/telegraf_plugins.yml:2580]: Not sure if the meaning is preserved. The plugin creates one file _per group_ and then writes all metrics belonging to this group to the respective file...

Comment (jstirnaman) [reply data/telegraf_plugins.yml:2580]: > Not sure if the meaning is preserved. The plugin creates one file _per group_ and then writes all metrics belonging to this group to the respective file...

Got it - updated.

**Review by jstirnaman — APPROVED** [review]

**Comment by srebhan** [comment]
@jstirnaman anything I can do to get this merged? The release is out since over a week now...

**Comment by jstirnaman** [comment]
> @jstirnaman anything I can do to get this merged? The release is out since over a week now...

Apologies! I didn't realize you were waiting on us.

**Pull request merged**